docker/generate_nibetaseries_images.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -e
+
+# Generate Dockerfile.
+generate_docker() {
+  docker run --rm kaczmarj/neurodocker:master generate docker \
+    --base=neurodebian:stretch-non-free \
+    --install gcc g++ \
+    --pkg-manager=apt \
+    --user=neuro \
+    --miniconda \
+      conda_install="python=3.6" \
+      pip_install="ipython nipype==1.1.5 pybids==0.6.4 nibabel==2.3.0 nistats==0.0.1b
```

Comment:
I think it will be more manageable if `nibetaseries` manages dependencies, since if the dependencies change in the project, then this will be "yet another file" that needs to be changed to keep up. To run the example, it looks like people would need `jupyter seaborn nibetaseries`.

Reply:
had another thought to make this more similar to fmriprep. fmriprep [copies the repository](https://github.com/poldracklab/fmriprep/blob/d29902e2a1e03840dc0c0e55e9dedd0d2dc30fca/Dockerfile#L178) into the docker image so that testing can occur within the docker image. so all that is needed from the conda environment is `jupyter seaborn`

Reply:
thanks for your input @jdkent, I implemented the requested changes.